include/zephyr/dt-bindings/pinctrl/nrf-pinctrl.h

```diff
@@ -172,34 +172,34 @@
 #define NRF_FUN_GRTC_CLKOUT_FAST 55U
 /** GRTC slow clock output */
 #define NRF_FUN_GRTC_CLKOUT_32K  56U
-/** SDP_MSPI CK */
-#define NRF_FUN_SDP_MSPI_SCK 57U
-/** SDP_MSPI DQ0 */
-#define NRF_FUN_SDP_MSPI_DQ0 58U
-/** SDP_MSPI DQ1 */
-#define NRF_FUN_SDP_MSPI_DQ1 59U
-/** SDP_MSPI DQ2 */
-#define NRF_FUN_SDP_MSPI_DQ2 60U
-/** SDP_MSPI DQ3 */
-#define NRF_FUN_SDP_MSPI_DQ3 61U
-/** SDP_MSPI DQ4 */
-#define NRF_FUN_SDP_MSPI_DQ4 62U
-/** SDP_MSPI DQ5 */
-#define NRF_FUN_SDP_MSPI_DQ5 63U
-/** SDP_MSPI DQ6 */
-#define NRF_FUN_SDP_MSPI_DQ6 64U
-/** SDP_MSPI DQ7 */
-#define NRF_FUN_SDP_MSPI_DQ7 65U
-/** SDP_MSPI CS0 */
-#define NRF_FUN_SDP_MSPI_CS0 66U
-/** SDP_MSPI CS1 */
-#define NRF_FUN_SDP_MSPI_CS1 67U
-/** SDP_MSPI CS2 */
-#define NRF_FUN_SDP_MSPI_CS2 68U
-/** SDP_MSPI CS3 */
-#define NRF_FUN_SDP_MSPI_CS3 69U
-/** SDP_MSPI CS4 */
-#define NRF_FUN_SDP_MSPI_CS4 70U
+/** HPF_MSPI CK */
+#define NRF_FUN_HPF_MSPI_SCK 57U
```

Comment:
Do these symbols actually need to be renamed? They are used also for sQSPI, so the HPF_ prefix in this context will be quite confusing. Wouldn't the current form be acceptable for HPF purposes? Alternatively, the indexes could be used twice, like:
```
#define NRF_FUN_SDP_MSPI_SCK 57U
#define NRF_FUN_HPF_MSPI_SCK NRF_FUN_SDP_MSPI_SCK
```
though this looks a bit awkward.